Makefile

```diff
@@ -67,19 +67,25 @@ help: ## This help.
 # -- Build --
 
 .PHONY: build
-build: get $(BINDIR)/$(BINNAME) ## Build go code
+build: get $(BINDIR)/$(BINNAME) # Build go code
 
 $(BINDIR)/$(BINNAME): $(SRC)
 	go build -tags excludecodegen,excludedist -ldflags '$(LDFLAGS)' -o $(BINDIR)/$(BINNAME) ./cmd/${BINNAME}
 	mkdir -p $(GOPATH)/bin/
 	cp $(BINDIR)/$(BINNAME) $(GOPATH)/bin/
 
+.PHONY: build-translate
```

Comment:
We need to run as dist with build-cross, so that binary is created for all platforms too. Do we want to name it as kubectl-translate, so that even in user's laptop it runs as a plugin.

Reply:
done.

Reply:
yes, build command should put it in path. line 75 does it for example